x-pack/plugins/cloud_security_posture/server/plugin.ts

```diff
@@ -117,5 +130,14 @@ export class CspPlugin
 
     return {};
   }
+
   public stop() {}
+
+  initialize(core: CoreStart): Promise<unknown> {
```

Comment:
The return value contains an `unknown` type. Why is that?

Reply:
fixed